daprdocs/content/en/reference/components-reference/supported-bindings/cosmosdb.md

```diff
@@ -56,6 +56,31 @@ This component supports **output binding** with the following operations:
 
 - `create`
 
+## Best Practices for Production Use
+
+Azure Cosmos DB shares a strict metadata request rate limit across all databases in a single Azure Cosmos DB account. New connections to Azure Cosmos DB assume a large percentage of the allowable request rate limit. (See the [CosmosDB documentation](https://docs.microsoft.com/azure/cosmos-db/sql/troubleshoot-request-rate-too-large#recommended-solution-3))
+
+Therefore several strategies must be applied to avoid simultaneous new connections to Azure Cosmos DB:
+
+- Ensure sidecars of applications only load the Azure Cosmos DB component when they require it to avoid unnecessary database connections. This can be done by [scoping your components to specific applications]({{< ref component-scopes.md >}}#application-access-to-components-with-scopes).
+- Choose deployment strategies that sequentially deploy or start all applications to minimize bursts in new connections to your Azure Cosmos DB accounts.
```

Comment:
```suggestion
- Choose deployment strategies that gradually deploy or start your applications to minimize bursts in new connections to your Azure Cosmos DB accounts.
```

Reply:
It actually is sequentially -- as in, deploy one application after the other. Gradually sounds to ambiguous / weak. It was a deliberate choice I made -- was leaning to `gradually` initially also :)